gpytorch/kernels/kernel.py

```diff
@@ -23,44 +23,51 @@
 from ..priors import Prior
 
 
-def default_postprocess_script(x):
-    return x
-
-
+def _sq_dist(x1, x2, x1_eq_x2=False):
```

Comment:
I saw two examples of these functions being used in BoTorch, so we could have these as public as well. I don't know to what extent other users are relying on `Distance`, but we could remove it from this commit without an internal change in the functionality, and the required changes to BoTorch would be minimal.